Themis/Models/Assessment.swift

```diff
@@ -102,6 +108,7 @@ struct AssessmentFeedback: Identifiable, Hashable {
         self.assessmentType = assessmentType
         self.type = type
         self.file = file
+        self.positive = true
```

Comment:
What is this used for and why is it set to true for this init?

Reply:
As far as I could understand from testing 1-2 hours with postman it identifies status of automatic test cases (wether they were successful or not). For manual test cases this is always true, which is why it's true by default. 
Technically I would just leave it out, but then automatic test cases will not be displayed in the web app anymore.